iron-checked-element-behavior.html

```diff
@@ -83,16 +88,15 @@
       }
     },
 
-    /**
```

Comment:
Comment is still good (or if it isn't, you should fix it, not delete it 😄)! We try to comment protected behaviour methods...